snap/snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: ros2-teleop
-version: '0.1'
+adopt-info: ros2-teleop
```

Comment:
ros2-teleop comes from a series of different source codes we need to version it differently. we had a discussion about versioning and for ros2-teleop we could live jazzy/humble and so on.